Source/Forms/Models/Gen4/WildModel4.cpp

```diff
@@ -284,7 +293,14 @@ QVariant WildSearcherModel4::data(const QModelIndex &index, int role) const
         case 3:
         {
             u8 item = state.getItem();
-            return item < 45 ? "No" : item < 95? "50%" : "5%";
+            if (state.getLead() == Lead::CompoundEyes)
+            {
+                return item < 20 ? "No" : item < 80? "60%" : "20%";
+            }
+            else
+            {
+                return item < 45 ? "No" : item < 95? "50%" : "5%";
+            }
```

Comment:
Same as above

Reply:
Leave the numbers then. Change `"No"` to `tr("None")`

Reply:
done